src/helm/benchmark/metrics/safety_metrics.py

```diff
@@ -0,0 +1,57 @@
+from typing import List
+
+from helm.benchmark.adaptation.adapter_spec import AdapterSpec
+from helm.benchmark.adaptation.request_state import RequestState
+from helm.benchmark.metrics.basic_metrics import compute_request_state_metrics
+from helm.benchmark.metrics.efficiency_metrics import EfficiencyMetric
+from helm.benchmark.metrics.metric import Metric
+from helm.benchmark.metrics.metric_name import MetricName
+from helm.benchmark.metrics.metric_service import MetricService
+from helm.benchmark.metrics.statistic import Stat
+
+
+class SafetyBasicGenerationMetric(Metric):
+    """Replacement for BasicGenerationMetric for HELM Safety.
+    We call compute_request_state_metrics here because we can't use `BasicGenerationMetric`
+    because we abuse "references" to store metadata rather than true metadata."""
```

Comment:
Oh no...  Can we just add a `metadata` field and not do this?

Reply:
Needs to be revisited after initial HELM safety results